assets/src/edit-story/output/story.js

```diff
@@ -49,16 +49,16 @@ function OutputStory({
         <CustomCSS />
         {/* Everything between these markers can be replaced server-side. */}
         <meta name="web-stories-replace-head-start" />
-        <link rel="canonical" href={story.link} />
+        <link rel="canonical" href={link} />
         <meta name="web-stories-replace-head-end" />
       </head>
       <body>
         <amp-story
           standalone="standalone"
           publisher={publisher.name}
-          publisher-logo-src={publisher.logo}
-          title={story.title}
-          poster-portrait-src={story.posterPortraitUrl || fallbackPoster}
+          publisher-logo-src={logoPlaceholder}
```

Comment:
Is this still valid AMP when it's just a placeholder?

Reply:
Nope, we can use WP fallback logo instead of that to make it valid but always static value. Just answered in Slack, too :)

Reply:
Changed in 1b012f3, PTAL